### ISSUE TITLE
TX does not activate after TX button click

### ISSUE THREAD

**Issue by vikulin** [jocover/ExtIO_HackRF#6]
TX does not activate after TX button click. Used latest ExtIO dll.

Configuration: Windows 7x64, HackRF One.

**Comment by vsboost** [comment]
Even though TX support is enabled via ExtIO, the problem is that HDSDR doesn't support the type of transmission.  I have confirmed this with the creator of HDSDR and he tells me that HDSDR requires some work to get this going.

**Comment by vikulin** [comment]
The http://www.hdsdr.de/faq.html tells opposite answer:

Q: TX doesn't work! How do I enable it?
A: Yes, this can happen. We extended the ExtIO-DLL interface to allow RX/TX switching in HDSDR versions 2.13 and above. TX functionality requires support from your ExtIO-DLL. Without this ExtIO support, HDSDR cannot provide the TX function. You may be able to get TX support for HDSDR by contacting the manufacturer of your radio, or alternatively by contacting the developer of the ExtIO.

I used HDSDR v 2.70 that is greater then mentioned 2.13 supporting TX.

Are sure that the TX does not work at all?

**Comment by vsboost** [comment]
Yes, below is an email from the maker of HDSDR only a couple of weeks ago when I asked him the same question.

"Tx support requires quite some work in HDSDR and in the DLL .. and i don't have an HackRF."

**Comment by vikulin** [comment]
He could contact with me and I could run his code on my HackRF. I'm ready to test this.

**Comment by vsboost** [comment]
good luck

lc@hdsdr.de

**Comment by Radiomix2000** [comment]
Any progress with TX support in HDSDR?

**Comment by jocover** [comment]
new header file have tx API ,but HDSDR  2.76 not support
http://www.hdsdr.de/download/LC_ExtIO_Types.h

> // preliminary TX function - not really tested!: lack of test hw
// status:
// 0: Samples with numIQsamples > 0 (=512) and non-NULL pointer interleavedIQ
// 1: Reset (to suspend/stop TX thread of HDSDR) with numIQsamples == 0 and interleavedIQ == NULL
// 2: Pause/Stop (buffer underrun) with numIQsamples == 0 and interleavedIQ == NULL
// 3: Continue (after buffer underrun) with numIQsamples == 0 and interleavedIQ == NULL
typedef void    (__stdcall * pfnTxSamples)      (int status, int numIQsamples, const short * interleavedIQ);

**Comment by Radiomix2000** [comment]
Ok. That's clear.
Do you plan do add HackRF Bias-T support to EXTIO?
Probably I can help you as much as allows my knowledge in programming.
But I need your assistance to help me to import the project to Visual Studio
